fe/fe-core/src/test/java/com/starrocks/analysis/ShowProcStmtTest.java

```diff
@@ -0,0 +1,56 @@
+// This file is made available under Elastic License 2.0.
+// This file is based on code available under the Apache license here:
+//   https://github.com/apache/incubator-doris/blob/master/fe/fe-core/src/test/java/org/apache/doris/analysis/DescribeStmtTest.java
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.